Makefile

```
@@ -113,13 +113,17 @@ docstyle:	## Check the docstring style using Docstyle checker
 ruff:	## Check source code using Ruff linter
 	uv run ruff check . --per-file-ignores=tests/*:S101 --per-file-ignores=scripts/*:S101
 
+lint-openapi: ## Lint docs/openapi.json (Spectral OAS ruleset; fail on error)
+	npx --yes @stoplight/spectral-cli@6 lint docs/openapi.json --fail-severity error --display-only-failures
+
 verify:	## Run all linters
```

checkmake (0.2.2)

[warning] Target body for "verify" exceeds allowed length of 5 (7).

(maxbodylength)